### ISSUE TITLE
Typings export

### ISSUE THREAD

**Issue by stellarhoof** [marnusw/date-fns-tz#209]
Do we need to include `typings.d.ts` in the package.json [exports](https://github.com/marnusw/date-fns-tz/blob/master/package.json#L26)? Typescript cannot find them even when I do `import 'date-fns-tz/typings.d.ts'` unless I manually add them there.

**Comment by laverdet** [comment]
Yes the `package.json` and build settings need to be updated for compatibility with ES Modules. The `exports` fields should look like:

```json
"./format": {
  "require": "./format/index.js",
  "types": "./format/index.d.ts",
}
```

..and so on.

You can workaround it in the meantime by adding a .d.ts file to your project with content [update the relative path as needed]:
```ts
/// <reference path="../node_modules/date-fns-tz/typings.d.ts" />
```
